es/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,257 +1,257 @@
-**1. Machine Learning tips and tricks cheatsheet**
+**1. Consejos y trucos sobre Aprendizaje Automático**
 
-&#10230;
+&#10230; Consejos y trucos sobre Aprendizaje Automático
 
 <br>
 
 **2. Classification metrics**
 
-&#10230;
+&#10230; Métricas para clasificación
 
 <br>
 
 **3. In a context of a binary classification, here are the main metrics that are important to track in order to assess the performance of the model.**
 
-&#10230;
+&#10230; En el contexto de una clasificación binaria, estas son las principales métricas que es importante seguir para evaluar el rendimiento del modelo.
```

Comment:
I would change singular "es" for plural "son" as the sentence is referring to the metrics (plural).